lib/Core/SpecialFunctionHandler.cpp

```diff
@@ -44,108 +44,103 @@ using namespace llvm;
 using namespace klee;
 
 namespace {
-  cl::opt<bool>
-  ReadablePosix("readable-posix-inputs",
-            cl::init(false),
-            cl::desc("Prefer creation of POSIX inputs (command-line arguments, files, etc.) with human readable bytes. "
-                     "Note: option is expensive when creating lots of tests (default=false)"));
+cl::opt<bool>
+ReadablePosix("readable-posix-inputs", cl::init(false),
+              cl::desc("Prefer creation of POSIX inputs (command-line "
+                       "arguments, files, etc.) with human readable bytes. "
+                       "Note: option is expensive when creating lots of tests "
+                       "(default=false)"));
 
-  cl::opt<bool>
-  SilentKleeAssume("silent-klee-assume",
-                   cl::init(false),
-                   cl::desc("Silently terminate paths with an infeasible "
-                            "condition given to klee_assume() rather than "
-                            "emitting an error (default=false)"));
+cl::opt<bool>
+SilentKleeAssume("silent-klee-assume", cl::init(false),
+                 cl::desc("Silently terminate paths with an infeasible "
+                          "condition given to klee_assume() rather than "
+                          "emitting an error (default=false)"));
 }
 
-
 /// \todo Almost all of the demands in this file should be replaced
 /// with terminateState calls.
 
 ///
 
-
```

Comment:
Add these lines again.

Reply:
this is the format tool deleted, do you really want to add again?

Reply:
noted.